exposed-tests/src/test/kotlin/org/jetbrains/exposed/sql/tests/shared/DDLTests.kt

```diff
@@ -353,7 +353,7 @@ class DDLTests : DatabaseTestsBase() {
             } get (t.id)
 
             val id2 = t.insert {
-                it[t.b] = longBlob
+                it[t.b] = blobParam(longBlob)
```

Comment:
Unless this fixes a particular bug, I would recommend creating a new test for that feature.

Reply:
It does not. Testing everything related to blobs in the `testBlob` case is pretty consistent with the rest of the test cases.

Reply:
@spand  It can still be in `testBlob`, but separate from the `longBlob` insert test. What do you think?

Reply:
I have created another statement for the blotParam case.